x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -121,11 +121,9 @@ export const updateAlertStatusAction = async ({
 export const determineToAndFrom = ({ ecs }: { ecs: Ecs[] | Ecs }) => {
   if (Array.isArray(ecs)) {
     const timestamps = ecs.reduce<number[]>((acc, item) => {
-      if (item.timestamp != null) {
-        const dateTimestamp = new Date(item.timestamp);
-        if (!acc.includes(dateTimestamp.valueOf())) {
-          return [...acc, dateTimestamp.valueOf()];
-        }
+      const dateTimestamp = new Date(item.timestamp ?? '');
```

Comment:
is it ok that `new Date('')` returns an `Invalid Date`?

Reply:
nope